### ISSUE TITLE
Translation namespaces

### ISSUE THREAD

**Issue by jim-taylor-business** [LemmyNet/lemmy-ui-leptos#27]
in a previous code review @dessalines mentioned that name-spacing did not turn out well for a previous UI project:

https://github.com/LemmyNet/lemmy-ui-leptos/pull/20/files#r1346348901

just wanted to highlight this so we can come to a consensus openly

**Comment by dessalines** [comment]
I would vote for **not** using name-spaces: we did this with jerboa, and very much regret it, because now we have 9 different strings keys for things like "go back".

cc @Baptistemontan @SleeplessOne1917

**Comment by SleeplessOne1917** [comment]
> [W]e have 9 different strings keys for things like "go back".

Maybe we could have a common namespace for things like that?

**Comment by dessalines** [comment]
I really don't get the rationale behind name-spacing translations at all. A translation is the same everywhere, it doesn't matter where it exists in the UI. If its to prevent large translation files, that's a style preference that to me is much worse than having a single file where you can see all your strings, like android does it with `strings.xml`

**Comment by Baptistemontan** [comment]
I'm with whatever you all are confortable with, my PR was more a suggestion than "this is the best you must do it".
I get your point, and translations are hard to deal with in general, I would have not proposed it in the first place but saw a PR that removed the `nav_` prefix to the translations keys, so I thought it was a good idea to use namespaces, or subkeys.

**Comment by jim-taylor-business** [comment]
my initial concern was the linguistic implications but from evidence from other projects it sounds like that is not a valid concern

my only other concern was code management/translation management  - is it easier to find and change translations if they are organised by context? again it sounds like this is not an issue in other projects. in fact it would introduce the complication of moving translations to the common namespace when they are found to be used in more than one place.

so in summary i am happy to go without namespaces

**Comment by jim-taylor-business** [comment]
would you like to make the changes for this one @Baptistemontan ? getting the translations working the way we want is very important, i think.

i'm happy to do it if you would like.

**Comment by Baptistemontan** [comment]
I'm not at home right now, if you want to do it be me guest.
